src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/DefaultLaunchProfileProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.IO;
+using System.ComponentModel.Composition;
+using ExportOrder = Microsoft.VisualStudio.ProjectSystem.OrderAttribute;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    [Export(typeof(IDefaultLaunchProfileProvider))]
+    [AppliesTo(ProjectCapability.LaunchProfiles)]
+    [ExportOrder(int.MinValue)] 
```

Comment:
use `Order.Default` instead of `int.MinValue`.

Reply:
Fixed